db/migrate/20230603232606_add_deck_nrdb_id.rb

```diff
@@ -0,0 +1,5 @@
+class AddDeckNrdbId < ActiveRecord::Migration[7.0]
+  def change
+    add_column :decks, :nrdb_id, :integer
```

Comment:
Please use the string UUID, not the integer ID.

Reply:
Done